tests/characters/test_bob_handles_frags.py

```diff
@@ -269,5 +259,24 @@ def test_bob_gathers_and_combines(enacted_federated_policy, federated_bob, feder
     # At long last.
     cleartext = federated_bob.verify_from(the_data_source, the_message_kit,
                                           decrypt=True,
-                                          delegator_signing_key=federated_alice.stamp.as_umbral_pubkey())
+                                          delegator_verifying_key=federated_alice.stamp.as_umbral_pubkey())
     assert cleartext == b'Welcome to the flippering.'
+
+
+def test_federated_bob_retrieves(federated_bob,
+                                 federated_alice,
+                                 capsule_side_channel,
+                                 ):
+
+    # The side channel is represented as a single MessageKit, which is all that Bob really needs.
```

Comment:
This comment appears not to be true - the next line suggests that it's a tuple of a `MessageKit` and an Enrico.